src/mcp_gsuite/tools_gslide.py

```
@@ -0,0 +1,1492 @@
+                    "find_text": {
+                        "type": "string",
+                        "description": "The text to find"
+                    },
+                    "replace_text": {
+                        "type": "string",
+                        "description": "The text to replace it with"
+                    },
+                    "match_case": {
+                        "type": "boolean",
+                        "description": "Whether to match case",
+                        "default": false
```

Ruff (0.8.2)

Undefined name `false`

(F821)